exercises/practice/binary-search/.docs/instructions.md

```diff
@@ -1,35 +1,29 @@
 # Instructions
 
-Implement a binary search algorithm.
+Your task is to implement a binary search algorithm.
 
-Searching a sorted collection is a common task. A dictionary is a sorted
-list of word definitions. Given a word, one can find its definition. A
-telephone book is a sorted list of people's names, addresses, and
-telephone numbers. Knowing someone's name allows one to quickly find
-their telephone number and address.
+A binary search algorithm finds an item in a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
+It allows us to quickly narrow down the possible locations of our item until we find it, or until we've eliminated all possible locations.
 
-If the list to be searched contains more than a few items (a dozen, say)
-a binary search will require far fewer comparisons than a linear search,
-but it imposes the requirement that the list be sorted.
+```exercism/caution
+Binary search only works when a list has been sorted.
+```
```

Comment:
```suggestion
<!-- prettier-ignore -->
~~~~exercism/caution
Binary search only works when a list has been sorted.
~~~~
```

This allows us to use `~~~~`. Note that under normal markdown rules, ` is the same as ~, hence it's reverting it each time.